.github/workflows/docker_test.yml

```diff
@@ -27,15 +43,23 @@ jobs:
     name: Test docker images
     steps:
       - uses: actions/checkout@v4
```

Comment:
I think you'll need to check out the specific tag from pypi here. See https://github.com/langflow-ai/langflow/blob/main/.github/workflows/docker-build.yml#L102

If not, you'll be checking out the default tag, which is `github.ref` (the ref from which the workflow is called). If that's the case, say you check out `main`. Then when you get the `expected_version` later, you may get `1.0.19`, but you ran the workflow from the nightly release and expect a version like`1.0.19.dev30`.